ARExperiment/ARExperiment/SizeComparisonViewController.swift

```diff
@@ -3,17 +3,61 @@ import SceneKit
 import ARKit
 
 class SizeComparisonViewController: UIViewController, ARExperimentSessionHandler {
+
+    enum CubeModel {
+        case big    //5x5
+        case medium //1x1
+        case small  //0.1x0.1
+        
+        var filename: String {
+            switch self {
+            case .big:
+                return "cube_5"
+            case .medium:
+                return "cube_1"
+            case .small:
+                return "cube_0.1"
+            }
+        }
+        
+        var textFilename: String {
+            switch self {
+            case .big:
+                return "text_5"
+            case .medium:
+                return "text_1"
+            case .small:
+                return "text_0.1"
+            }
+        }
+    }
     
+    enum Scale {
```

Comment:
Similar comment from above for the Scale.

Reply:
Ok